app/main.py

```diff
@@ -1 +1,37 @@
-#  write your code here
+class Animal:
+    def __init__(self, name: str,
+                 appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
```

Comment:
Can you make this method static ? I am about `@staticmethod` : ) If you do not using `self` in your function it is better to use decorator  `@staticmethod`